src/state/state.ts

```diff
@@ -2,19 +2,42 @@ import type { BigNumber } from 'ethers';
 import { produce, type Draft } from 'immer';
 
 import type { Config } from '../config/schema';
-import type { LocalSignedData, AirnodeAddress, TemplateId } from '../types';
+import type { chainId, DApiName, DecodedDataFeed, DataFeedId, SignedData } from '../types';
 
 interface GasState {
   gasPrices: { price: BigNumber; timestampMs: number }[];
   sponsorLastUpdateTimestampMs: Record<string, number>;
 }
 
+export interface DataFeedOnChainValue {
+  value: BigNumber;
+  timestamp: number; // in seconds
```

Comment:
```suggestion
  timestamp: number;
```

We already have a strong convention that timestamp is in seconds. Whenever there is ms timestamp (except maybe in tests) there should be a suffix `Ms` in the variable name